x-pack/plugins/security/server/authorization/actions/alerting.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { isString } from 'lodash';
+
+export class AlertingActions {
```

Comment:
This will be confusing for the Alerting folks: this is *not* our `actions` this is the *security* `actions` which are basically a sort of URI describing a privilege which is made up of the version, scope, entity and operation.

Reply:
Seems like it's worth adding a comment in the code for this